api-reference/10 UI Widgets/dxFileManager/1 Configuration/itemView/details/details.md

```diff
@@ -4,7 +4,8 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures the "Details" file system representation mode.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+[note] The [itemView.mode](/api-reference/10%20UI%20Widgets/dxFileManager/1%20Configuration/itemView/mode.md '/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/itemView/#mode') option should be set to **details**. 
```

Comment:
We usually give such things in the short description, like in [this article](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#allowSorting), because this way the reader have no chance of missing this information.